microsoft-edge/devtools-guide-chromium/whats-new/2022/07/devtools-104.md

```diff
@@ -0,0 +1,113 @@
+---
+title: What's new in DevTools (Microsoft Edge 104)
+description: "Use Layers in 3D View. Improved display of request URLs in Issues tool. Improving accessibility for the Detached Elements tool. Fix: Arrows representing web socket traffic now render correctly in the Network tool. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/02/2022
+---
+# What's New in DevTools (Microsoft Edge 104)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Use Layers in 3D View
+
+<!-- Title: Where did Layers go? -->
+<!-- Subtitle: Layers isn't a stand-alone tool anymore, it's wrapped up in the comprehensive 3D View; a tool for all your visual debugging needs. -->
+
+The standalone **Layers** tool has been removed in Microsoft Edge version 104.  In previous versions of Microsoft Edge, the **Layers** tool redirected users to the **3D View** tool.
+
+In the **3D View** tool, the **Composited Layers** view has all the features you love from the **Layers** tool, along with some updates.  The **Composited Layers** view is now complemented by two other visual debugging panes: **DOM** and **Z-index**.  Combine these additional views with the **Composited Layers** view for better debugging.
+
+![The Composited Layers tab in the 3D View tool](./devtools-104-images/3d-view-composited-layers.png)
+
+See also:
+* [Debug the web in 3D with the 3D View tool](https://blogs.windows.com/msedgedev/2022/06/21/debug-the-web-in-3d-with-the-3d-view-tool/)
+* [Navigate webpage layers, z-index, and DOM using the 3D View tool](../../../3d-view/index.md)
+
+
+<!-- ====================================================================== -->
+## Request URLs in Issues tool have been shortened to display better
+
```

Comment:
Title should be "In Issues tool, long request URLs have been shortened" as shown in the metadata below.

Reply:
Made heading match "Title:" comment tho usu want to be different